test/GuardClauses.UnitTests/GuardAgainstExpression.cs

```diff
@@ -26,60 +26,49 @@ public static IEnumerable<object[]> GetCustomStruct()
     [InlineData(10)]
     public void GivenIntegerWhenTheExpressionEvaluatesToTrueDoesNothing(int test)
     {
-        Guard.Against.AgainstExpression((x) => x == 10, test, "Value is not equal to 10");
+        Guard.Against.Expression((x) => x == 10, test, "Value is not equal to 10");
     }
 
     [Theory]
     [InlineData(10)]
     public void GivenIntegerWhenTheExpressionEvaluatesToFalseThrowsException(int test)
     {
-        Assert.Throws<ArgumentException>(() => Guard.Against.AgainstExpression((x) => x == 5, test, "Value is not equal to 10"));
+        Assert.Throws<ArgumentException>(() => Guard.Against.Expression((x) => x == 5, test, "Value is not equal to 10"));
     }
 
     [Theory]
     [InlineData(1.1)]
     public void GivenDoubleWhenTheExpressionEvaluatesToTrueDoesNothing(double test)
     {
-        Guard.Against.AgainstExpression((x) => x == 1.1, test, "Value is not equal to 1.1");
+        Guard.Against.Expression((x) => x == 1.1, test, "Value is not equal to 1.1");
```

Comment:
Same feedback here. Use a [Fact] and note that value *cannot* be 1.1 in this case.

Reply:
Thank you for the feedback! Made two more commits addressing the issues.